docs/index.yml

```diff
@@ -64,10 +64,10 @@ conceptualContent:
       links:
         - url: core/introduction.md
           itemType: overview
-          text: .NET Core overview
-        - url: core/whats-new/dotnet-core-3-1.md
+          text: .NET overview
+        - url: core/whats-new/dotnet-five.md
           itemType: whats-new
-          text: What's new in .NET Core 3.1
+          text: What's new in .NET 5
         - url: ./core/introduction.md
```

Comment:
Tour of .NET and .NET overview seem to be pointing to the same article.

Reply:
Yeah, unsure where these would go. I'll leave them the same for now.

Reply:
I would delete the Tour of .NET.